.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+_Thanks for wanting to report an issue you've found in Node.js. Please delete
+this text and fill in the template below. If unsure about something, just do as
+best as you're able._
+
+_Note that it will be much easier for us to fix the issue if a test case that
+reproduces the problem is provided. Ideally this test case should not have any
+external dependencies. We understand that it is not always possible to reduce
+your code to a small test case, but we would appreciate to have as much data as
+possible. Thank you!_
```

Comment:
It is 5 sentences in total, including `Thank you!` :) I guess we will probably stick to this until later time!